src/components/Form/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
```

Comment:
Do we still need this component? If so, can you specify where are we going to use it?

Reply:
No need for this component. I thought it comes when I rebase ``development`` and forget to delete it.